docs/webusb.md

```diff
@@ -0,0 +1,35 @@
+Zephyr.js API for WebUSB
+========================
+
+* [Introduction](#introduction)
+* [Class: WebUSB](#class-webusb)
+* [API Documentation](#api-documentation)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+The WebUSB module supports advertising the device as a WebUSB device when the
+USB port is connected to a PC. Currently, this only works on Arduino 101 as it
+is the only board with a USB driver in the Zephyr tree.
+
+The API allows you to set the URL that will be suggested to the browser for
+connecting to your device.
+
+When you connect your device to a Linux PC or Mac with Chrome >= 60 running, it
+will give a notification that the device would like you to visit the URL you've
+set. (Windows currently prevents this from working, I believe.)
```

Comment:
@larsgk will know the details

Reply:
M61 (stable, sans flag, Windows:no pop-up yet and Android: likewise but slightly different ). Btw - I'd love to get it on the FRDM-KL25Z target as it has a separate USB port that works very well in our ARMmbed WebUSB setups (now that A101 was mentioned)